core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java

```diff
@@ -26,6 +26,18 @@ public void testProcessQuery() throws Exception {
         assertEquals("The oncogenicity is not empty, but it should.", "", indicatorQueryResp.getOncogenic());
         assertTrue("There is treatment(s) in the response, but it should no have any.", indicatorQueryResp.getTreatments().size() == 0);
 
+        query = new Query(null, null, null, "CD74-CD74", null, "structural_variant", StructuralVariantType.DELETION, "Pancreatic Adenocarcinoma", null, null, null, null);
```

Comment:
Looking at the issue associated with this PR, I wonder should you test "Bladder Urothelial Carcinoma" instead of "Pancreatic Adenocarcinoma"?

Reply:
Tumor type does not matter in this case